src/utils/sendUpdates.ts

```diff
@@ -102,10 +103,19 @@ export default class Updates {
         messageUpdates.channelType,
       )
       response.forEach(async (activity: wikiActivities) => {
+        // SEND UPDATE ON DISCORD
         messageUpdates.channelId.send({
           embeds: [await this.messageWikiStyle(activity, messageUpdates.url)],
         })
+
+        // SEND UPDATE ON TWITTER
         await this.checkAndTweet(messageUpdates, activity)
+
+        // REVALIDATE WIKI PAGE
+        await this.wikiUpdates.revalidateWikiPage(
```

Comment:
we might want to try catch this in case there is an error. we shouldnt stop the process because of a revalidation issue

Reply:
Ohh Thanks ! i will add a new commit for this